modules/odc_con_customizing-a-developer-catalog-or-its-sub-catalogs.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: CONCEPT
+
+[id="odc_con_customizing-a-developer-catalog-or-its-sub-catalogs_{context}"]
+= Developer catalog and sub-catalog customization
+
+As a cluster administrator, you have the ability to organize and manage the Developer catalog or its sub-catalogs by enabling or disabling the sub-catalog types or disabling the entire developer catalog. In some situations, customers do not want their users to have access to the Developer catalog or one or more of its sub-catalogs.
```

Comment:
This sentence is a bit long - maybe split it:

As a cluster administrator, you have the ability to organize and manage the Developer catalog or its sub-catalogs.
You can enable or disable....

Reply:
Fixed. Also, I am removing the line, "In some situations...." since after review it does not add anything to the content.